importer/importer.go

```diff
@@ -44,6 +44,27 @@ func (imp *dbImporter) ImportBlock(blockbytes []byte) (txCount int, err error) {
 	return imp.ImportDecodedBlock(&blockContainer)
 }
 
+func countInnerAndAddParticipation(stxns []types.SignedTxnWithAD, participants [][]byte) (int, [][]byte) {
+	copyAddr := func(addr types.Address) []byte {
+		return append([]byte(nil), addr[:]...)
+	}
+	num := 0
+	for _, stxn := range stxns {
+		participants = participate(participants, copyAddr(stxn.Txn.Sender))
```

Comment:
why copying address?

Reply:
It was being overwritten by the range

Reply:
Not sure I understand. `stxn` - yes, it is overwritten and that's why it is an error to take a loop var address.
In the same time referencing to values is fine - they either fully copied (primitives) or shallow-copied and point to different underlying memory in case of slices/maps, like in this example https://play.golang.org/p/6yT1jFVJc3s